apps/academy-wep/components/learn/resource-card.tsx

```
@@ -2,118 +1,129 @@
-      <div className="absolute inset-0 bg-gradient-to-br from-gray-50 via-white to-gray-50 opacity-0 group-hover:opacity-100 transition-opacity duration-500"></div>
+	return (
+		<motion.div
+			key={resource.id}
+			initial={{ opacity: 0, y: 20 }}
+			animate={{ opacity: 1, y: 0 }}
+			exit={{ opacity: 0, y: -20 }}
+			transition={{ duration: 0.4, delay: index * 0.05 }}
+			className="group relative overflow-hidden bg-white rounded-2xl transition-all duration-500 hover:shadow-xl flex flex-col h-full border border-gray-100 shadow-sm"
+		>
+			{/* Hover effect background */}
+			<div className="absolute inset-0 bg-gradient-to-br from-gray-50 via-white to-gray-50 opacity-0 group-hover:opacity-100 transition-opacity duration-500"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)